packages/stargate/src/queries/index.ts

```diff
@@ -9,3 +9,4 @@ export { BankExtension, setupBankExtension } from "./bank";
 export { DistributionExtension, setupDistributionExtension } from "./distribution";
 export { IbcExtension, setupIbcExtension } from "./ibc";
 export { setupStakingExtension, StakingExtension } from "./staking";
+export { createRpc } from "./utils";
```

Comment:
`createPagination` would be nice to have available in other packages as well. We created many of those objects by hand in cosmwasm-stargate.